themes/src/main/resources-community/theme/base/login/messages/messages_es.properties

```diff
@@ -156,6 +230,7 @@ invalidPasswordMinSpecialCharsMessage=Contrase\u00F1a incorrecta: debe contener
 invalidPasswordNotUsernameMessage=Contrase\u00F1a incorrecta: no puede ser igual al nombre de usuario.
 invalidPasswordRegexPatternMessage=Contrase\u00F1a incorrecta: no cumple la expresi\u00F3n regular.
 invalidPasswordHistoryMessage=Contrase\u00F1a incorrecta: no puede ser igual a ninguna de las \u00FAltimas {0} contrase\u00F1as.
+invalidPasswordGenericMessage=Contrase\u00F1a no v\u00E1lida: La nueva contrase\u00F1a no cumple con las politicas de contrase\u00F1a.
```

Comment:
"pol\u00EDticas"

Also there's another style inconsistency here. Some translations say "no v\u00E1lida" (not valid), while others say "inv\u00E1lida" (invalid)